app/_locales/es/messages.json

```diff
@@ -1,10 +1,840 @@
+  },
+  "spaceBetween": {
+    "message": "Solo puede haber un espacio entre las palabras"
+  },
+  "loweCaseWords": {
+    "message": "palabras semilla solo puede tener minusculas"
+  },
+  "seedPhraseReq": {
+    "message": "palabras semilla tienen 12 palabras de largo"
+  },
+  "addTokens": {
+    "message": "Agregar tokens"
+  },
+  "addCustomTokens": {
+    "message": "Agregar custom token"
+  },
+  "up": {
+    "message": "arriba"
+  },
+  "down": {
+    "message": "abajo"
+  },
+  "tokenWarning1": {
+    "message": "Mantenga un registro de los tokens que ha comprado con su cuenta de MetaMask. Si compraste tokens usando una cuenta diferente, esos tokens no aparecerán aquí."
+  },
+  "tokenSelection": {
+    "message": "Busca tokens o seleccionalo de nuestra lista de tokens populares."
+  },
+  "search": {
+    "message": "Buscar."
+  },
+  "privacyMsg": {
+    "message": "Politica de privacidad"
+  },
+  "terms": {
+    "message": "Terminos de Uso"
+  },
+  "attributions": {
+    "message": "Atribuciones"
+  },
+  "supportCenter": {
+    "message": "Visita nuestro centro de atencion"
+  },
+  "knowledgeDataBase": {
+    "message": "Visita nuestra base de conocimiento"
+  },
+  "visitWebSite": {
+    "message": "Visita nuestro sitio web"
+  },
+  "followTwitter": {
+    "message": "Siguenos en Twitter"
+  },
+  "emailUs": {
+    "message": "Envianos un correo!"
+  },
+  "hereList": {
+    "message": "Aqui esta una lista!!!"
+  },
+  "insufficientFounds": {
+    "message": "Fondos insuficientes."
+  },
+  "insufficientTokens": {
+    "message": "Tokens insuficientes."
+  },
+  "negativeETH": {
+    "message": "No se pueden mandar cantidades negativas de ETH."
+  },
+  "urlErrorMsg": {
+    "message": "URIs necesita el HTTP/HTTPS prefijo apropiado"
+  },
+  "invalidRPC": {
+    "message": "Invalida URL del RPC"
+  },
+  "saveLogs": {
+    "message": "Logs de estado contienen tus direcciones publicas y transacciones enviadas"
+  },
+  "stateLogs": {
+    "message": "Logs de estado"
+  },
+  "downloadStatelogs": {
+    "message": "Descargar logs de estados"
+  },
+  "revealSeedWorld": {
+    "message": "Revelar palabras de semilla"
+  },
+  "revealSeedWorldWarning": {
+    "message": "No recuperes tu semilla en un lugar publico! Esas palabras pueden ser usadas para robarte todas tus cuentas"
+  },
+  "resetAccount": {
+    "message": "Resetar cuenta"
+  },
+  "californiaRoll": {
+    "message": "Metamask fue diseñado y construido en California.  "
+  },
+  "classicInterface": {
+    "message": "Usar interfaz clasica. "
+  },
+  "welcomeMeta": {
+    "message": "Bienvenido a Metamask Beta."
+  },
+  "metaIs": {
+    "message": "Metamask is una identidad segura para Ethereum"
+  },
+  "holdEther": {
+    "message": "Te permite mantener tus ether y tokens,y te sirve como puente para aplicaciones desentralizadas. "
+  },
+  "decimalsMustZerotoTen": {
+    "message": "Los decimales deben ser al menos 0 y no más de 36."
```

Comment:
`Los decimales deben ser al menos 0 y no más de 36` Let's remove the periods for consistence.

Reply:
What do you mean ? @hermanjunge

Reply:
Sáquemos los puntos al final de las oraciones. Eso.